src/app/backend/integration/api/types.go

```diff
@@ -24,6 +24,7 @@ type IntegrationID string
 // Integration app IDs should be registered in this block.
 const (
 	HeapsterIntegrationID IntegrationID = "heapster"
```

Comment:
Do we plan to remove Heapster soon?

Reply:
I was thinking that we'd have one or two releases that supported both Heapster and Metrics Server, then remove Heapster support entirely.

If consensus is to rip the bandaid off and remove Heapster support with this PR, I can do that pretty easily. Just let me know. :)

Reply:
@floreks WDYT?

Reply:
I'd do as @jeefy suggests. Leave for a couple of releases until heapster is completely removed.